src/stock-reports/generate-report/create-stock-report.scss

```diff
@@ -1,15 +1,19 @@
 @use '@carbon/styles/scss/spacing';
 @use '@carbon/styles/scss/type';
```

Comment:
```suggestion
```

Prefer the shorter imports from the `layout` and `type` namespaces on lines 3-4.